toolkit/tools/pkg/imagecustomizerlib/customizeos.go

```diff
@@ -75,6 +75,11 @@ func doOsCustomizations(buildDir string, baseConfigPath string, config *imagecus
 		return err
 	}
 
+	err = cleanUpImage(imageChroot)
```

Comment:
One of the policies of MIC is that we don’t do anything unless the user explicitly asks for something. As such any clean-up operations should be attached to the operations that made the leftover artifacts.

In this case, the tdnf cleanup code should be moved under `customizepackages.go` and should only be done if we installed or updated packages.

Reply:
I see your point, but then a customer doing a tdnf install from a script will see a difference in image size and verity hash. Do we need to add explicit documentation on that behavior?

Reply:
@cwize1 thoughts?

Reply:
If the user uses `tdnf` in a script, then they can call `tdnf clean all` as well.